### DISCUSSION TITLE
shift-click

### DISCUSSION THREAD

**Discussion by kenNash** [johnlindquist/kit#777] [Errors]
Shift-clicking the tray icon in the menu bar I get "command click" (not "shift click").
Is something wrong with me?

(John, thanks for support and dedication!)

**Comment by johnlindquist** [comment]
Good catch! Try and spot the bug 😅 
![CleanShot 2022-06-01 at 13 10 30](https://user-images.githubusercontent.com/36073/171483452-e965cb46-804a-468e-95e5-15b21a700d93.png)

**Reply by kenNash** [reply]
🤣
where is this code from?

**Reply by johnlindquist** [reply]
The `Kit.app` is in another repo:
https://github.com/johnlindquist/kitapp/blob/main/src/tray.ts#L35

(I pushed a build out last night that fixed this)
